scipy/stats/_continuous_distns.py

```diff
@@ -7489,11 +7489,10 @@ def fit_loc_scale_w_shape_gt_1():
 
             # if the sign of `dL_dLocation_star` is positive at rbrack,
             # we're not going to find the root we're looking for
-            i = 1
             delta = (data.min() - rbrack)
             while dL_dLocation_star(rbrack) > 0:
-                rbrack = data.min() - i * delta
-                i *= 2
+                rbrack = data.min() - delta
+                delta *= 2
```

Comment:
The original error was that `i` got too big. This can be avoided if we just double `delta`.